HW04/Pavel_Mishchenko/spec/mentor_spec.rb

```diff
@@ -0,0 +1,210 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+
+RSpec.describe Mentor do
+  let(:student) { Student.new(name: 'TestName', surname: 'TestSurname') }
+  let(:mentor) { described_class.new(name: 'Test', surname: 'Testov') }
+  let(:notification1) { Notification.new(title: 'TestTitle1', description: 'TestDescription1') }
+  let(:notification2) { Notification.new(title: 'TestTitle2', description: 'TestDescription2') }
+
+  describe '#notify_student' do
+    context "when method hasn't arguments" do
+      subject { mentor.notify_student }
+
+      it 'raises ArgumentError' do
+        expect { subject }.to raise_error(ArgumentError)
```

Comment:
as you use subject, you can read about [such construction](https://relishapp.com/rspec/rspec-core/docs/subject/one-liner-syntax)

Reply:
The one-liner syntax only works with non-block expectations (e.g. expect(obj).to eq, etc) and it cannot be used with block expectations (e.g. expect { object }).

Reply:
but there are other situations, where is_expected can be used